pkgs/by-name/lz/lzbench/package.nix

```
@@ -27,6 +27,7 @@ stdenv.mkDerivation rec {
     description = "In-memory benchmark of open-source LZ77/LZSS/LZMA compressors";
     license = licenses.free;
```

```suggestion
    license = with licenses; [ gpl2Only gpl3Only ];
```

https://github.com/inikep/lzbench/pull/158

While you are at it